app/src/main/java/sdp/moneyrun/Riddle.java

```diff
@@ -2,29 +2,28 @@
 
 import java.util.Arrays;
 import java.util.HashSet;
+import java.util.Objects;
 import java.util.Set;
 
 public class Riddle {
 
-    private final String question;
-    private final String correctAnswer;
-    private final String firstAnswer;
-    private final String secondAnswer;
-    private final String thirdAnswer;
-    private final String fourthAnswer;
-
-
+    //They should be final, but that is incompatible with being able to add them to the DB
+    private String question;
+    private String correctAnswer;
+    private String firstAnswer;
+    private String secondAnswer;
+    private String thirdAnswer;
+    private String fourthAnswer;
 
     /**
      * @param question        This is the question that the user will see and will have to solve
      * @param correctAnswer   This is the unique correct answer to the question
      */
     public Riddle(String question, String correctAnswer, String firstAnswer, String secondAnswer, String thirdAnswer, String fourthAnswer) {
-        if (question == null || correctAnswer == null)
+        if (question == null || correctAnswer == null || firstAnswer == null ||
```

Comment:
Revert to what was here previously, the aim was to remove code smells.

Reply:
Done, it didn't seem like a necessary change at the time.

Reply:
I agree I did the same as you in the first place but it's simply to avoid having a "bad score" in code climate.